### DISCUSSION TITLE
jgit.ssh.apache update sshd version

### DISCUSSION THREAD

**Discussion by dengmh123** [eclipse-jgit/jgit#75] [Q&A]
hello, due to CVE-2022-45047 , i want to update jgit.ssh.apache(5.13.3.202401111512-r) dependency about org.apache.sshd sshd-sftp to 2.10.0. but i found it can't run in this case, can you help me how to update.
just like this 

    <dependencies>
        
        <dependency>
            <groupId>org.eclipse.jgit</groupId>
            <artifactId>org.eclipse.jgit.ssh.apache</artifactId>
            <version>5.13.3.202401111512-r</version>
            <exclusions>
                <exclusion>
                    <groupId>org.apache.sshd</groupId>
                    <artifactId>sshd-sftp</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.apache.sshd</groupId>
                    <artifactId>sshd-osgi</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <dependency>
            <groupId>org.apache.sshd</groupId>
            <artifactId>sshd-sftp</artifactId>
            <version>2.10.0</version>
        </dependency>
        <dependency>
            <groupId>org.apache.sshd</groupId>
            <artifactId>sshd-osgi</artifactId>
            <version>2.10.0</version>
        </dependency>
    </dependencies>

**Comment by msohn** [comment]
5.13 is EOL. We don't have enough developers to continue maintenance of branch stable-5.13.
You should rather update to the latest release 6.10.0.202406032230-r.
Next planned release is 7.0.0 which is [planned to be released on Sep 11](https://projects.eclipse.org/projects/technology.jgit/governance).
